python/lsst/ts/ATHexapod/model.py

```diff
@@ -32,37 +25,19 @@ def is_connected(self):
         connected : bool
 
         """
-        return self.reader is not None and self.writer is not None
+        return self.controller.is_connected()
```

Comment:
You have many methods that you have duplicated from controller (is_connected, write_command, real_position , etc.). In fact I do not see the difference between Model and Controller. Do you need both classes?

I *strongly* urge you to refactor this code to only have one class.

Or, if you want to keep both classes, then please refactor the code to make it clear what each class does and why they are separate. Once you decide the responsibility of each class then I hope you will get rid of all Model methods that just call the same Controller method (have the Model user talk to the `controller` attribute directly) and/or move some of the methods from `Controller` to `Model` if that makes more sense.